lib/vdc/DCMPAS.cpp

```diff
@@ -1156,28 +1156,49 @@ int DCMPAS::_InitCoordvars(
 int DCMPAS::_InitDerivedVars(
 	NetCDFCollection *ncdfc
 ) {
+
 	int rc = _InitVerticalCoordinatesDerivedAtmosphere(ncdfc);
 	if (rc<0) return(-1);
 
 	rc = _InitVerticalCoordinatesDerivedOcean(ncdfc);
 	if (rc<0) return(-1);
 
-    // Create and install the Time coordinate variable
-    //
-	DerivedCoordVar_WRFTime *derivedVar = new DerivedCoordVar_WRFTime(
-        timeDimName, ncdfc, xTimeVarName, timeDimName
-    );
+	DC::CoordVar cvarInfo;
+	if (ncdfc->VariableExists(ncdfc->GetNumTimeSteps()-1, xTimeVarName)) {
 
-	rc = derivedVar->Initialize();
-	if (rc<0) return(-1);
+		// Create and install the Time coordinate variable
+		//
+		DerivedCoordVar_WRFTime *derivedVar = new DerivedCoordVar_WRFTime(
```

Comment:
Did you intend to use the derived WRF time here as opposed to the new derived time class?

Reply:
Yes, the new class is used below.